doc/tutorials/cloud/00_setup.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>I think, we are supposed to be downloading from S3. I need to check which URL to use.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>We can fetch them from maven, but github releases are easier to work with (api for getting the latest release, etc). I don't think we have our jar files on s3</p>

Reply:
<p>I think that is a topic we need to revisit later, for the moment it is ok. I first need to figure out what our current status is regarding S3.</p>